pkg/package.go

```diff
@@ -24,6 +25,7 @@ type PackageInfo struct {
 	Modularitylabel string
 	Summary         string
 	PGP             string
+	PackageDigest   string
```

Comment:
I want to use the same name in the rpmdb as much as possible.

```suggestion
	SigMD5   string
```

Reply:
Changed in https://github.com/knqyf263/go-rpmdb/pull/39/commits/34acf20e5c24e385837199d0e4d154927eba9860